health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/SearchFileIds.js

```
@@ -0,0 +1,73 @@
+        tenantId: Digit.ULBService.getCurrentTenantId(),
+        pageSize: "100",
+        uuid: uuids
+      },
+    });
+
+    // debugger;
+    if (response1?.user?.length === 0) {
+      throw new Error("No users found with the given uuid");
+    }
+    let uuidName={};
+    for (const ob of response1?.user){
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)